poetry.lock

```diff
@@ -1,31 +1,81 @@
+tests-mypy = ["mypy (>=1.6)", "pytest-mypy-plugins"]
+tests-no-zope = ["attrs[tests-mypy]", "cloudpickle", "hypothesis", "pympler", "pytest (>=4.3.0)", "pytest-xdist[psutil]"]
 
 [[package]]
 name = "bpy-addon-build"
-version = "0.2.1"
+version = "0.3.0"
 description = "A build system to make building and testing Blender addons 10 times easier"
 optional = false
-python-versions = ">=3.7,<4.0"
+python-versions = ">=3.8"
+files = [
+    {file = "bpy_addon_build-0.3.0-1-py3-none-any.whl", hash = "sha256:c97bcef8fe0f46fbd4448728ed39c8482864a69d71f46b0f47f596a6e6d8f1c2"},
+    {file = "bpy_addon_build-0.3.0-py3-none-any.whl", hash = "sha256:37e49f74c95955ced676f44f80510200a8e0f96664f71284766664f66a29b9cb"},
+    {file = "bpy_addon_build-0.3.0.tar.gz", hash = "sha256:d9fb05588611c57fd4c0ca2e9b78975514f437c004401cc0861bb746ad9ebb56"},
+]
+
+[package.dependencies]
+attrs = ">=23.1.0,<24.0.0"
+cattrs = ">=23.2.3,<24.0.0"
+pyyaml = ">=6.0.1,<7.0.0"
+rich = ">=13.7.0,<14.0.0"
+
+[[package]]
+name = "cattrs"
+version = "23.2.3"
+description = "Composable complex class support for attrs and dataclasses."
+optional = false
+python-versions = ">=3.8"
 files = [
-    {file = "bpy_addon_build-0.2.1-py3-none-any.whl", hash = "sha256:443a0df1e940070a956278040ab04774bcbf42090aba8a05240d7b51928e443f"},
-    {file = "bpy_addon_build-0.2.1.tar.gz", hash = "sha256:175e2afad291d1def1612b5e55ce70fe686f9a108f9f78f5d823e4c3cd6745ae"},
+    {file = "cattrs-23.2.3-py3-none-any.whl", hash = "sha256:0341994d94971052e9ee70662542699a3162ea1e0c62f7ce1b4a57f563685108"},
+    {file = "cattrs-23.2.3.tar.gz", hash = "sha256:a934090d95abaa9e911dac357e3a8699e0b4b14f8529bcc7d2b1ad9d51672b9f"},
 ]
 
 [package.dependencies]
-docopt = ">=0.6.2,<0.7.0"
-pyyaml = ">=6.0,<7.0"
-rich = ">=13.3.5,<14.0.0"
+attrs = ">=23.1.0"
+exceptiongroup = {version = ">=1.1.1", markers = "python_version < \"3.11\""}
+typing-extensions = {version = ">=4.1.0,<4.6.3 || >4.6.3", markers = "python_version < \"3.11\""}
+
+[package.extras]
+bson = ["pymongo (>=4.4.0)"]
+cbor2 = ["cbor2 (>=5.4.6)"]
+msgpack = ["msgpack (>=1.0.5)"]
+orjson = ["orjson (>=3.9.2)"]
+pyyaml = ["pyyaml (>=6.0)"]
+tomlkit = ["tomlkit (>=0.11.8)"]
+ujson = ["ujson (>=5.7.0)"]
 
 [[package]]
-name = "docopt"
-version = "0.6.2"
-description = "Pythonic argument parser, that will make you smile"
+name = "exceptiongroup"
+version = "1.2.0"
+description = "Backport of PEP 654 (exception groups)"
 optional = false
-python-versions = "*"
+python-versions = ">=3.7"
 files = [
-    {file = "docopt-0.6.2.tar.gz", hash = "sha256:49b3a825280bd66b3aa83585ef59c4a8c82f2c8a522dbe754a8bc8d08c85c491"},
+    {file = "exceptiongroup-1.2.0-py3-none-any.whl", hash = "sha256:4bfd3996ac73b41e9b9628b04e079f193850720ea5945fc96a08633c66912f14"},
+    {file = "exceptiongroup-1.2.0.tar.gz", hash = "sha256:91f5c769735f051a4290d52edd0858999b57e5876e9f85937691bd4c9fa3ed68"},
 ]
 
+[package.extras]
+test = ["pytest (>=6)"]
```

Comment:
Where is pytest being used? Is this something pulled in by one of the dependent libraries? MCprep itself is using the built in unittest library.

Reply:
Might be from when I initially had `polib` as one of the dependencies defined in `pyproject.toml`. I'll regenerate the lock file

Reply:
Ok I've regenerated the file and it still has it. My best guess is that it comes from `flake8` (which uses pytest). I think it should be fine to keep it though, as it shouldn't interfere with what we have.

Reply:
Gotcha, thanks for checking - I'm fine to leave it as-is then